core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -78,6 +78,9 @@
   {% if not iframed %}
     <attribution-guide></attribution-guide>
   {% endif %}
+  {% if nav_mode in ['explore'] %}
+    <exploration-footer></exploration-footer>
+            {% endif %}
```

Comment:
I think the thing to do here might actually be to leave this file alone, and in exploration_player.html, do:

```
{% block footer %}
  <exploration-footer></exploration-footer>
{% endblock footer %}
```

Try it and see? The idea is to replace the footer block in base.html with something custom.